src/libical/vcomponent_cxx.cpp

```diff
@@ -207,7 +207,7 @@ ICalProperty *VComponent::get_next_property(const icalproperty_kind &kind)
 }
 
 /* Working with components */
-/* Return the first VEVENT, VTODO or VJOURNAL sub-component if it is one of those types */
+/* Returns the first VEVENT, VTODO or VJOURNAL sub-component if it is one of those types */
```

Comment:
@ Returns pls

Reply:
Should I do this for comments like this with only one sentence? That looks a little weird imo.

Reply:
I kinda prefer "@return" instead of "@returns" 
doesn''t matter that much to me.  and we are using @return in many, many places already.
up to you're willingness to do the work.

Reply:
Ok, I think I might leave it as is. This is big enough already.